tests/lang_callable/macros/tmacros_various.nim

```diff
@@ -328,3 +330,16 @@ block: # bug #15118
 
   block:
     flop("b")
+
+block: # Ensure nkCommentStmt equality is not ignored when checking for const duplicates:
```

Comment:
Sorry for being daft, is there a duplicate constant value check, and here we're ensuring that it's not triggered?

Also is that an error or a warning?

Reply:
There's a duplicate constant value check in `vmgen.nim` for literals, `cmpNodeCnst` for NimNode literals, that deduplicates constants as vm code is generated. It used to not account for differing `nkCommentStmt`s so it counted the second "comment 2" as a duplicate of "comment 1"

Reply:
Ah, got it, some of the above explanation making it into the above comment might help, it's a bit arcane -- soft suggestion.

Reply:
Added the explanation to the test in ae58d49dd051593a2155edc350923a5fe412f99c